recipes/plaster_pastedeploy/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "plaster_pastedeploy" %}
+{% set version = "0.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c231130cb86ae414084008fe1d1797db7e61dc5eaafb5e755de21387c27c6fae
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
```

Comment:
Seems you are missing the [entry_points](https://github.com/Pylons/plaster_pastedeploy/blob/fa9fe09d6f74630e033f86e1de5dfe43d36e025b/setup.py#L54-L67).

See [Python entry_points](https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points).

Reply:
Added but not sure if it is correct because in setup.py has duplicated entries.